chromium_src/components/content_settings/core/browser/content_settings_registry.cc

```diff
@@ -393,6 +393,12 @@ void ContentSettingsRegistry::BraveInit() {
           WebsiteSettingsRegistry::PLATFORM_ANDROID |
           WebsiteSettingsRegistry::PLATFORM_IOS,
       WebsiteSettingsInfo::INHERIT_IN_INCOGNITO);
+  website_settings_registry_->Register(
+      ContentSettingsType::BRAVE_PSST, brave_shields::kBravePsst, base::Value(),
+      WebsiteSettingsInfo::UNSYNCABLE, WebsiteSettingsInfo::NOT_LOSSY,
+      WebsiteSettingsInfo::REQUESTING_SCHEMEFUL_SITE_ONLY_SCOPE,
```

Comment:
this is per-frame and site instead of origin? That doesn't sound right?

Reply:
also everywhere else you're using origin so this definitely seems like the wrong option here